src/frameworks/fastify.js

```diff
@@ -10,6 +17,12 @@ function getValidator(validateRequest) {
 
     function validate(request, reply) {
         const requestOptions = _getParameters(request);
+        if (skiplist.some((skipListRegex) => {
+            return skipListRegex.test(requestOptions.path);
+        })) {
+            return Promise.resolve();
```

Comment:
wouldn't you prefer explicitly return false?

Reply:
I checked express and koa implementations, they do not return boolean when validation passes without errors, they simply return `undefined`. Hence it makes sense to do the same thing here. `Promise.resolve` is important to make sure that we are returning thenable Promise in all cases ,though.